src/ServiceBus.AttachmentPlugin/AzureStorageAttachmentConfiguration.cs

```diff
@@ -33,7 +33,7 @@ public class AzureStorageAttachmentConfiguration
             string blobEndpoint,
             string containerName = "attachments",
             string messagePropertyToIdentifyAttachmentBlob = "$attachment.blob",
-            Func<Message, bool> messageMaxSizeReachedCriteria = null)
+            Func<Message, bool>? messageMaxSizeReachedCriteria = null)
```

Comment:
mixing  `= null` and `= default` (above)?

Reply:
I have to admit that `= default` still feels weird. Though you're right, I should stick to a single way and `= default` is probably the right way to follow. Will update.

Reply:
Looks like R# doesn't like `Func<Message, bool>?`
![image](https://user-images.githubusercontent.com/1309622/68618659-10f7e900-0487-11ea-9886-1c894bc06b92.png) ATM it's filed as opened https://github.com/dotnet/roslyn-analyzers/issues/2877
Have you encountered that?